src/amo/components/RatingManager/index.js

```diff
@@ -147,12 +147,13 @@ export class RatingManagerBase extends React.Component<Props, State> {
         return i18n.gettext('Log in to rate this theme');
       case ADDON_TYPE_EXTENSION:
         return i18n.gettext('Log in to rate this extension');
-      default:
-        if (!validAddonTypes.includes(addonType)) {
-          throw new Error(`Unknown extension type: ${addonType}`);
-        }
-        log.warn(`Using generic prompt for add-on type: ${addonType}`);
+      default: {
+        const validAddonType = validAddonTypes.includes(addonType);
+        log.warn(`Using generic prompt for ${validAddonType ?
+          'add-on' : 'unknown extension'} type: ${addonType}`
```

Comment:
so if the addon type is invalid, it will print `using generic prompt for unknown extension`, but is it really an extension? Why do you want to update the log message here?

Reply:
well that's what the message said before basically - in that it referred to it as an extension if it was not valid : `Unknown extension type: ${addonType}`. Since we are using a prompt now (instead of throwing the error), I really just added this to the beginning of the original text

Reply:
mmh but this does not look correct to me, because add-on is the most generic term if I am not mistaken (i.e. a web extension is an add-on).

Reply:
I see what you're saying...so what about what this:
```javascript
   log.warn(`Using generic prompt for ${validAddonType ?
     'add-on' : 'unknown addon-on'} type: ${addonType}`
```

Reply:
:+1: perfect!